perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -216,7 +216,7 @@ hashref.
 
 sub base_url_for_report {
     my ( $self, $report ) = @_;
-    return $self->base_url;
+    return base_url_with_lang($self);
```

Comment:
Thanks for this – this should be `$self->base_url_with_lang` so that if the function is subclassed the correct one will be called.